release/release_after_kokoro.sh

```diff
@@ -20,6 +20,11 @@ elif [ ! -x "$ECLIPSE_HOME/eclipse" ]; then
     exit 1
 fi
 
+if ! `which -s xml`; then
```

Comment:
Hmm... `which` on my machine doesn't accept `-s`. :disappointed: (For this version, the man page looks like this: https://www.techonthenet.com/linux/commands/which.php.)

Perhaps `which xmlstarlet > /dev/null`?